frigate/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: frigate_web_1
+      APP_PORT: 5000
+      PROXY_AUTH_ADD: "false"
+  
+  web:
+    privileged: true # this may not be necessary for all setups
+    network_mode: host
+    restart: unless-stopped
+    image: ghcr.io/blakeblackshear/frigate:stable@sha256:bb7f7e76a13eccef0b12704e5851cc774a12af1f12df387d6a70a796a3e938c3
+    shm_size: "128mb" # update for your cameras based on calculation above
+    devices:
+     # - /dev/bus/usb:/dev/bus/usb # passes the USB Coral, needs to be modified for other versions
+     # - /dev/dri/renderD128 # for intel hwaccel, needs to be updated for your hardware
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - ${APP_DATA_DIR}/config.yml:/config/config.yml
+      - ${APP_DATA_DIR}/storage:/media/frigate
+      - type: tmpfs # Optional: 1GB of memory, reduces SSD/SD Card wear
+        target: /tmp/cache
+        tmpfs:
+          size: 1000000000
+    ports:
+      - "8554:8554" # RTSP feeds
+      - "8555:8555/tcp" # WebRTC over tcp
+      - "8555:8555/udp" # WebRTC over udp
```

Comment:
If we are in host network mode then we can remove this entirely, but we'll have to document carefully the full set of ports that are exposed to avoid current and future apps from clashing.

Do you know if it's binding these ports like this is all that is necessary? Or would users need to port forward from their router if they are behind a firewall?

Reply:
`network_mode: host` removed.